{"name": "test_api2[王五-30]", "status": "broken", "statusDetails": {"message": "NameError: name 'YamlUtil' is not defined", "trace": "@pytest.fixture(scope=\"session\", autouse=True)\n    def clear_yaml():\n>       YamlUtil().clear_extract_yaml()\nE       NameError: name 'YamlUtil' is not defined\n\nconftest.py:14: NameError"}, "parameters": [{"name": "name", "value": "'王五'"}, {"name": "age", "value": "30"}], "start": 1669275299212, "stop": 1669275299212, "uuid": "095dd0e4-579f-4c41-be81-df8207521e72", "historyId": "96a4b27e5ee45ba68c4a014be8466241", "testCaseId": "a7b6fe78ffeca118517d048a45bbe491", "fullName": "testcase.test _api.TestApi#test_api2", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test _api"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "admin"}, {"name": "thread", "value": "55928-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test _api"}]}